### ISSUE TITLE
Docstring for function with decorators (@property) not displayed in IntelliSense

### ISSUE THREAD

**Issue by Filip-Tomasko** [microsoft/pylance-release#696]
## Environment data

-   VS Code version: 1.51.1
-   Extension version: 2020.11.371526539
-   OS and version: Windows 10 20H2
-   Python version: 3.9.0

## Expected behaviour

IntelliSense should show the docstring for a `@property` function when hovering over it, but doesn't.
It should display something like this:
![](https://i.imgur.com/C1CFjo3.png)

## Actual behaviour

It displays just its type.
![](https://i.imgur.com/iB56XHE.png)

## Steps to reproduce:

```Python
class MyClass(object):
    """
    MyClass Docstring
    """

    def __init__(self, length):
        self._length = length

    @property
    def length(self):
        """
        This should be displayed, but isn't
        """
        return self._length

    @length.setter
    def length(self, value):
        if value < 0:
            raise ValueError("The length cannot be negative!")
        else:
            self._length = value


one = MyClass(3)
one.length = -2
```

1. Put this code into VS Code
2. Hover over `length` in `one.length` in the last line
3. See that it displays `length: int` instead of `This should be displayed, but isn't`

##

Maybe I'm just doing it wrong, if so please enlighten me. (I'm learning Python and it's my first time using `@property`)

**Comment by erictraut** [comment]
It doesn't appear that you're using Pylance. Perhaps you have MPLS or Jedi configured instead?

Here's what Pylance displays in this case:
![Screen Shot 2020-12-07 at 10 47 40 AM](https://user-images.githubusercontent.com/7040122/101391951-ee96f180-3879-11eb-99db-f031923b4044.png)

**Comment by Filip-Tomasko** [comment]
Seems like @erictraut was right: Jedi was selected as the Python language server
![](https://i.imgur.com/QvgwOiW.png)

I installed Pylance as @luabud suggested and now it displays properly like in the screenshot above. I guess I'll use Pylance from now on. 

Though I'm not sure if that's the optimal outcome. But thanks for helping me :D

**Comment by Filip-Tomasko** [comment]
I just noticed one thing. I don't know if that's desired behavior or not, but in the autocompletion tooltip the docstring for the property is not displayed, only for the class.
![image](https://user-images.githubusercontent.com/3923334/101422588-4de71700-38f7-11eb-9cf0-7a79d8c29865.png)
![image](https://i.imgur.com/xQKPyAT.png)

**Comment by jakebailey** [comment]
Does it work if you complete it and hover over `length`, i.e. does it only break when it's in the completion tooltip?

**Comment by Filip-Tomasko** [comment]
Indeed. That's the case. I could make a video/gif showcasing that if you need it

**Comment by savannahostrowski** [comment]
This will be fixed in the next version of Pylance.

@bschnurr

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.2.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202120-3-february-2021
